locales/en-US/app.ftl

```diff
@@ -66,6 +66,12 @@ CallNodeContextMenu--transform-merge-call-node = Merge node only
         function’s node that called it. It only removes the function from that
         specific part of the tree. Any other places the function was called from
         will remain in the profile.
+CallNodeContextMenu--transform-merge-unaccounted-native-functions = Merge unaccounted native frames
+    .title =
+        Removes stack frames such as unsymbolicated JIT frames from the call tree,
+        and make their callers absorb their cost.
+        Specifically, this transform merges any native stack frames which were
```

Comment:
I'm a bit lost on the second part (mind you, not a developer here).

Is this transform merging unaccounted for frames to a shared library (the action is "merge to a shared library"), or is this merging frames not belonging to a shared library ("merge specific frames, i.e. those that don't belong to a library")? Also, what does unaccounted for mean here: not examined, not included, something else?